### ISSUE TITLE
Not working with dbt Snowflake adapter

### ISSUE THREAD

**Issue by dweaver33** [yu-iskw/action-sqlfluff#16]
When I run this action with the following setup:
```yaml
 name: Lint files
        id: lint-files
        uses: yu-iskw/action-sqlfluff@v3
        with:
          github_token: ${{ secrets.github_token }}
          reporter: github-pr-review
          sqlfluff_command: "lint" # Or "lint"
          config: "${{ github.workspace }}/.sqlfluff"
          paths: '${{ github.workspace }}/models'
          templater: dbt
```

I get the following error:
```bash
 Running sqlfluff 🐶 ...
  19:42:09  Error importing adapter: No module named 'dbt.adapters.snowflake'
  Traceback (most recent call last):
    File "/usr/local/lib/python3.9/site-packages/dbt/config/profile.py", line 183, in _credentials_from_profile
      cls = load_plugin(typename)
    File "/usr/local/lib/python3.9/site-packages/dbt/adapters/factory.py", line 215, in load_plugin
      return FACTORY.load_plugin(name)
    File "/usr/local/lib/python3.9/site-packages/dbt/adapters/factory.py", line 70, in load_plugin
      raise RuntimeException(f'Could not find adapter type {name}!')
  dbt.exceptions.RuntimeException: Runtime Error
    Could not find adapter type snowflake!
```

I suspect this is because this action seems to install `dbt-core` and not the adapter specific dbt.

**Comment by yu-iskw** [comment]
@dweaver33 Thank you for the feedback. I will look into it.

**Comment by yu-iskw** [comment]
I understand the issue is caused by the lack of installation of `dbt-snowflake`. [`sqlfluff-template-dbt`](https://pypi.org/project/sqlfluff-templater-dbt/) tried to use a dbt profile based on the configuration, but `dbt-snowflake` can't be found. 

The issue probably haapens on any dbt adapter with `dbt-core>=1.0.0`. I will think of a way to install extra required packages for dbt adapters.

**Comment by adeoluwaakande** [comment]
I get a similar error with dbt-bigquery:

```
=== [dbt templater] Sorting Nodes...
  10:49:34  Error importing adapter: No module named 'dbt.adapters.bigquery'
  Traceback (most recent call last):
    File "/usr/local/lib/python3.9/site-packages/dbt/config/profile.py", line 183, in _credentials_from_profile
      cls = load_plugin(typename)
    File "/usr/local/lib/python3.9/site-packages/dbt/adapters/factory.py", line 215, in load_plugin
      return FACTORY.load_plugin(name)
    File "/usr/local/lib/python3.9/site-packages/dbt/adapters/factory.py", line 70, in load_plugin
      raise RuntimeException(f'Could not find adapter type {name}!')
  dbt.exceptions.RuntimeException: Runtime Error
    Could not find adapter type bigquery!
```

**Comment by yu-iskw** [comment]
@dweaver33 @adeoluwaakande 
I added a new action's input `extra_requirements_txt` to install your dbt adapters. As you can imagine, conflicts can occur, if the extra `requirements.txt` contains `sqlfluff` and its dependent modules. So, I would recommend you put only dbt adapter you use into it. The changes were released at v3.2.0. Please feel free to give me any feedback on it.

- https://github.com/yu-iskw/action-sqlfluff/pull/17
- https://github.com/yu-iskw/action-sqlfluff/releases/tag/v3.2.0
